### DISCUSSION TITLE
Cannot download PHP manager from windows website

### DISCUSSION THREAD

**Discussion by mathieu-theresa** [snipe/snipe-it#14641] [Q&A]
Hi all,

I'm following a tutorial to install snipe-it on a windows 2019 server. I need to download and install [PHP Manager 1.5.0 for IIS 10](https://www.iis.net/downloads/community/2018/05/php-manager-150-for-iis-10). I went to the Microsoft website but nothing happen when I click the download link !!

Can someone help me or share the file?

Thank you so much!

**Comment by Sh1aze** [comment]
Experiencing the same problem now 😅. Did you find anything that worked for you @mathieu-theresa? Can anyone help us in this issue, thanks!

**Comment by mathieu-theresa** [comment]
Yes, I found it. I don't remember where and it took me forever.
Here
<https://drive.google.com/file/d/1qyZMk_YTizMGJMVULN_TtCwVY9sxw9lz/view?usp=sharing>is
a Google Drive Link to the program.

I hope that helps. Good luck with the installation.

**Comment by Sh1aze** [comment]
Cool, thank you for this!! Have a nice day.

**Comment by jdorste** [comment]
Here's the latest MSI installer from the Github page that created the PHP Manager listed on the Microsoft site.

https://github.com/RonaldCarter/PHPManager/releases/tag/V1.5.0

I installed it and it works correctly on IIS 10.

To verify this is the original author, scroll down to the bottom of this page for the source docs: https://www.iis.net/downloads/community/2018/05/php-manager-150-for-iis-10

Good Luck :)

**Comment by snipe** [comment]
I think the standard PHP installation would be the best approach - https://www.php.net/manual/en/install.windows.iis.php

**Reply by jdorste** [reply]
Hi snipe. 

Thanks for the suggestion. What we are talking about is something a little different though.

After doing the standard PHP installation, there is a plugin or extension type thing for IIS called PHP Manager that helps you manage PHP after it is installed. PHP Manager helps you do things such as enabling, disabling, and adding php extensions, changing the PHP version, and other settings. See the attached screenshots.

This used to be available in the Web Platform Installer in IIS, which basically let you add plugins/extensions, but they got rid of the Web Platform Installer, and did not replace it with anything.

Let me know if you know of a better way. 

Thanks!

![PHP_Manager](https://github.com/user-attachments/assets/f32c1714-dc2c-45b3-84ca-0818719113ac)
![PHP_Manager2](https://github.com/user-attachments/assets/27ca247a-fb67-48ee-8a83-169426abb0fb)

**Reply by snipe** [reply]
It looks like according to their own forums, something like Chocolatey is recommended.

https://learn.microsoft.com/en-us/answers/questions/429383/web-platform-installer-end-of-support-and-sunsetti

**Reply by jdorste** [reply]
Interesting. I'll have to look into that. It's just really annoying that they killed the Web Platform Installer without a replacement... Ugh... 

Thanks!

**Reply by snipe** [reply]
Agreed :(

**Reply by marcusmoore** [reply]
Something else that might be helpful for anyone that stumbles upon this discussion: [How to install PHP on Windows using Winget](https://php.watch/articles/winget-install-php)

**Reply by indangerofcollapsing** [reply]
> It looks like according to their own forums, something like Chocolatey is recommended.
> 
> https://learn.microsoft.com/en-us/answers/questions/429383/web-platform-installer-end-of-support-and-sunsetti

Thank you for this - was trying to get my PHP working and it really helped. Thanks.
